printer_mods/SinisterRj/SafetyPrinter/BackConnector/readme.md

```diff
@@ -0,0 +1,21 @@
+<p>This is handfull if you use external sensors, like smoke detectors or an emergency stop button.</P>
```

Comment:
Please avoid using html in markdown files.  images and paragraphs have native markdown syntax.

Reply:
ok, I'll check this.